oauthlib/oauth2/rfc6749/grant_types/implicit.py

```diff
@@ -319,7 +335,7 @@ def validate_token_request(self, request):
         if request.response_type is None:
             raise errors.MissingResponseTypeError(request=request)
         # Value MUST be set to "token".
-        elif request.response_type != 'token':
+        elif not request.response_type in self.response_types:
```

Comment:
IIRC, the [OpenIDConnect spec allows](https://openid.net/specs/openid-connect-core-1_0.html#Authentication) specifying multiple, space-separated response types.

Should this line be something like:

`if not set(self.request.response_type.split()).issubset(self.response_types):`